internal/controllers/inventory_controller.go

```diff
@@ -1541,6 +1600,47 @@ func (t *reconcilerTask) deployServer(ctx context.Context, serverName string) (u
 		},
 	}
 
+	if utils.IsOAuthEnabled(t.object) && utils.RequiresInternalProxy(serverName) {
+		deploymentSpec.Template.Spec.Containers = append(deploymentSpec.Template.Spec.Containers, corev1.Container{
+			Name: utils.InternalRbacContainerName,
+			SecurityContext: &corev1.SecurityContext{
+				AllowPrivilegeEscalation: &privilegeEscalation,
+				Capabilities: &corev1.Capabilities{
+					Drop: []corev1.Capability{"ALL"},
+				},
+			},
+			Image:           os.Getenv(utils.KubeRbacProxyImageName),
+			ImagePullPolicy: corev1.PullPolicy(os.Getenv(utils.ImagePullPolicyEnvName)),
+			Args: []string{
+				fmt.Sprintf("--secure-listen-address=0.0.0.0:%d", utils.InternalProxyPort),
+				fmt.Sprintf("--upstream=http://127.0.0.1:%d/", utils.DefaultContainerPort),
+				"--logtostderr=true",
+				"--tls-cert-file=/secrets/tls/tls.crt",
+				"--tls-private-key-file=/secrets/tls/tls.key",
+				"--tls-min-version=VersionTLS12",
```

Comment:
Would it make sense to have `VersionTLS12` a constant since it's used in multiple places? Mostly for the case when we'll have to move to the next version.
Maybe even the `10` below?

Reply:
fixed